app/src/main/res/layout/activity_compose.xml

```diff
@@ -21,6 +21,34 @@
             android:padding="8dp"
             tools:ignore="ContentDescription" />
         <!--content description will be set in code -->
+
+        <Button
+            android:id="@+id/atButton"
+            style="?attr/image_button_style"
+            android:layout_width="?attr/actionBarSize"
+            android:layout_height="?attr/actionBarSize"
+            android:layout_gravity="end"
+            android:contentDescription="\@"
+            android:padding="8dp"
+            android:layout_marginHorizontal="4dp"
+            android:text="\@"
+            android:textColor="?android:textColorTertiary"
+            android:textSize="?attr/status_text_large"
+            />
```

Comment:
make both these Buttons `androidx.appcompat.widget.AppCompatButton`, width 40dp, textStyle bold.
`layout_marginHorizontal` only works on Api 26+

`contentDescription` should not be necessary when `text` is the same. Please extract the string resource in a string to make the warning go away (you can put it in donottranslate.xml)